server/lib/nl/config_builder/timeline.py

```diff
@@ -31,11 +31,25 @@
 
 # set the line tile spec field for a date range or a single date.
 def _set_line_tile_spec(date_range: types.Date, single_date: types.Date,
-                        line_tile_spec: LineTileSpec):
+                        line_tile_spec: LineTileSpec,
+                        sv_place_facet: types.Sv2Place2Facet):
   if date_range:
     start_date, end_date = get_date_range_strings(date_range)
```

Comment:
nit: Can we put date-range handling in a helper?

Reply:
done